formatting/goodFiles/source/README.md

```diff
@@ -0,0 +1,6153 @@
+/*
```

Comment:
Could this be a simple MD file with a small amount of improperly formatted code rather than an entire file from the kernel?